html/changelogs/example copy.yml

```diff
@@ -0,0 +1,58 @@
+################################
```

Comment:
Wrong file? Or at least it should be only one

Reply:
I have no idea why all this stuff is being added in - the commits from the other PR I did https://github.com/Aurorastation/Aurora.3/pull/19459 here